### ISSUE TITLE
Bad encoding configuration reading

### ISSUE THREAD

**Issue by dusan19** [quatanium/python-onvif#96]
I have a camera which has H265 encoding set. When i read the configuration with GetVideoEncoderConfigurations(), it shows that the H264 encoding is set.
How can i read and set the proper configuration now that onvif has support for H265 encoding?

**Comment by yingchengpa** [comment]
no. you should use the media2.wsdl(/ver20/media/wsdl/) to get the h265 encodeing.
